@noun-auction/components/ActiveAuction/ActiveAuctionRow.tsx

```diff
@@ -135,13 +201,20 @@ export function ActiveAuctionRow({
           </CollectionLink>
         )}
         {!useModal && <Separator mt="x1" />}
-        {timerComplete ? (
+        {auctionCompleted ? (
           <SettleAuction
+            auctionContractAddress={auctionContractAddress}
+            layout={layout === 'sideBarBid' ? 'row' : 'historyOnly'}
             useErrorMsg={useErrorMsg}
             className="nounish-auction__row-link"
           />
         ) : (
-          <PlaceNounsBid useModal={useModal} />
+          <PlaceNounsBid
+            layout={layout === 'sideBarBid' ? 'row' : 'historyOnly'}
```

Comment:
If we're repeating the layout logic let's just do it in one place and store it in a const?